src/test/java/example/Example.java

```diff
@@ -5,6 +5,7 @@
 package example;
```

Comment:
hrmm. yea, it's serving 2 purposes, but i think you make a good point that it could belong in the same package and having a different one doesn't buy much of anything. moving it to `com.palantir.websecurity.examples`